lib/jsonParser.go

```diff
@@ -195,26 +195,25 @@ func GetNameFromWritten(match MatchInfoRequest) string {
 
 // Data from pit scouting
 type PitScoutingData struct {
-	TeamNumber    int    `json:"Team"`    // The team number
-	PitIdentifier string `json:"Pit"`     // The pit identifier, as seen on the pit map
-	Scouter       string `json:"Scouter"` // The person who did the pit scouting
-
-	Drivetrain string       `json:"Drivetrain"` // The type of drivetrain the robot has
-	Distance   DistanceData `json:"Distance"`   // The information on the distance shooting it can do
-
-	AutoScores     int  `json:"Auto Scores"`  // The average scores this robot gets in auto
-	MiddleControls int  `json:"Middle Notes"` // The average number of notes from the middle this robot gets in auto
-	NoteDetection  bool `json:"Detection"`    // If it has note detection
-
-	Cycles           int             `json:"Cycles"`       // The average number of cycles it gets in teleop
-	DriverExperience int             `json:"Experience"`   // The years of experience the driver has
-	BotType          string          `json:"Bot Type"`     // The type of robot it is (offense, defense, amp, speaker, etc)
-	HumanPlayer      HumanPlayerData `json:"Human Player"` // Data regarding this team's human player
-
-	EndgameBehavior string  `json:"Endgame Behavior"` // What does this robot do in endgame? (Park, climb, trap, etc)
-	ClimbTime       float64 `json:"Climb Time"`       // How long does it take for this robot to climb?
-
-	Notes string `json:"Notes"` // Other notes
+	TeamNumber int `json:"Team"` // The team number
+	// PitIdentifier string `json:"Pit"`     // The pit identifier, as seen on the pit map
```

Comment:
Is this intentionally commented out, or a leftover from earlier? If the former, why?